src/state/slices/opportunitiesSlice/resolvers/foxFarming/index.ts

```diff
@@ -315,3 +318,11 @@ export const foxFarmingStakingUserDataResolver = async ({
 
   return { data }
 }
+
+export const foxFarmingLpOpportunityIdsResolver = (): Promise<{
+  data: GetOpportunityIdsOutput
+}> => Promise.resolve({ data: [...foxEthLpAssetIds] as LpId[] })
```

Comment:
Littlish hack to avoid this guy

```shell
Conversion of type 'readonly [AssetId]' to type 'LpId[]' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  The type 'readonly [AssetId]' is 'readonly' and cannot be assigned to the mutable type 'LpId[]'.
  ```

Reply:
This pattern worries me a little. 

I'm wondering if `LpId` should just be an alias for `AssetId` for now so we don't end up casting everywhere and then forget to fix the casts when an `LpId` can be something other than an `AssetId`...

Reply:
Agreed ! We actually discussed this one out with @0xdef1cafe as this is a lot of of overhead and possibly unsafe, see https://github.com/shapeshift/web/pull/3218/#discussion_r1021908858 as an example.
I'll tackle it today before wiring up Idle metadata / populating Idle user data